app/src/main/res/layout/fragment_browser_tab.xml

```diff
@@ -47,17 +47,25 @@
         tools:listitem="@layout/item_autocomplete_search_suggestion"
         tools:visibility="visible" />
 
-    <androidx.recyclerview.widget.RecyclerView
-        android:id="@+id/quickAccessSuggestionsRecyclerView"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:background="?attr/daxColorSurface"
+<!--    <androidx.recyclerview.widget.RecyclerView-->
```

Comment:
Delete?

Reply:
Done in https://github.com/duckduckgo/Android/pull/4568